spcconverter.qml

```diff
@@ -571,27 +571,55 @@ MuseScore {
         debugLog("Loaded parameter settings from score.");
     }
 
-    function processAnnotations(texts, staff) {
+    function getVolumeFromDynamic(dynamic) {
+        var volumeMap = {
+            // 255
+            "fff": "FF",
+            // 222
+            "ff": "DE",
+            // 186
+            "f": "BA",
+            // 150
+            "mf": "96",
+            "mp": "96",
+            // 108
+            "p": "6C",
+            // 72
+            "pp": "48",
+            // 36
+            "ppp": "24"
+        };
+        return volumeMap[dynamic];
+    }
+
+    function processAnnotations(annotations, staff) {
         var addSection = false;
         var sectionName = "";
         var prefix = "";
         var postfix = "";
-        for (var i = 0; i < texts.length; i++) {
+        for (var i = 0; i < annotations.length; i++) {
```

Comment:
Does QML support ECMAScript 2015? If so, I would _highly_ recommend using instead:
```suggestion
        for (var annotation of annotations) {
```
And then replace all instances of `annotations[i]` with `annotation`.